project/src/events/login.js

```diff
@@ -0,0 +1,10 @@
+const checkCredentials = function () {
+    const username = document.querySelector('#username').value;
+    const password = document.querySelector('#password').value;
+
+    if (username === 'user' && password === 'pass') {
+        goToEvaluationsPage();
+    } else {
+        alert('Username or Password wrong. Please try again!');
```

Comment:
a better user experience is to add a div in the page for errors display and style it with an error class. In which case you can also remove the no-alert from eslint.

Reply:
I'll try. I've seen eslint doesn't like alerts, but I didn't know why, :D